contracts/swappa/PairSavingsCELO.sol

```diff
@@ -33,5 +35,22 @@ contract PairSavingsCELO is ISwappaPairV1 {
     }
 	}
 
+	function getOutputAmount(
+		address input,
+		uint amountIn,
+		bytes calldata data
+	) external view override returns (uint amountOut) {
+		address savingsCELOAddr = parseData(data);
+		if (input == 0x2879BFD5e7c4EF331384E908aaA3Bd3014b703fA) {
+			// Savings CELO
+			return ISavingsCELO(savingsCELOAddr).savingsToCELO(amountIn);
```

Comment:
im not sure if this branch really even make sense, because `swap()` is 1-directional for this pair, we can only deposit Celo to get sCelo out, not the other way around.

I'm inclined to even just make `getOutputAmount` return 0 if the input token is `sCelo` because if sCelo is sent into this pair's Swap(), nothing will come out